shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -1920,8 +2066,38 @@ module MakeImpl<InputSig Lang> {
           )
         }
 
+        predicate callEdgeArgParam(
+          DataFlowCall call, DataFlowCallable c, ArgNodeEx arg, ParamNodeEx p,
+          boolean allowsFieldFlow, Ap ap
+        ) {
+          exists(FlowState state |
+            flowIntoCallAp(call, c, arg, p, ap) and
```

Comment:
Because that doesn't include flow-through edges, they were accounted for when the callable was entered, which in reverse flow means the out-to-return edge. Also, the restriction that this would achieve is already included in a stronger way by joining with the pair of taken-edge predicates.